.env.template

```
@@ -8,4 +8,8 @@ FLASK_DEBUG=true
 #MongoDB configuration
 MONGODB_CONNECTION_STRING=mongodb_connection_string
 MONGODB_DB_NAME=mongodb_db_name
-MONGODB_COLLECTION_NAME=mongodb_collection_name
\ No newline at end of file
+MONGODB_COLLECTION_NAME=mongodb_collection_name
+
+#OAuth configuration
+OAUTH_CLIENT_ID=flask_login_client_id
+OAUTH_CLIENT_SECRET=lask_login_client_secret
```

Not that it matters in the slightest but..

```suggestion
OAUTH_CLIENT_SECRET=flask_login_client_secret
```